src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -175,7 +175,8 @@ typedef struct wrapped_rr_closure_arg {
   /* The RR instance related to the closure */
   grpc_lb_policy* rr_policy;
 
-  /* The grpclb instance that created the wrapping */
+  /* The grpclb instance that created the wrapping. This instance is not owned,
+   * reference counts are untouched. Its used only for logging purposes. */
```

Comment:
s/Its/It's/

Reply:
Uuuugh, one of my pet peeves. The shame... fixed.